README.md

```diff
@@ -29,6 +29,7 @@ which is useful if you're trying the latest code from
       -V, --version         show program's version number and exit
       -u {now,daily,never,latest}, --update {now,daily,never,latest}
                             update schedule for NVD database (default: daily)
+      -n, --nvd             use nvd cve retrieval api
```

Comment:
```suggestion
      -n, --nvd  {api, json}           choose method for getting CVE lists from NVD
```

Actually, why don't we just go ahead and make it so that `--nvd=api` and `--nvd=json` are both explicit options with `json` as the default for now.  Then, assuming we don't have any issues and the performance seems good, we could plan to make `--nvd=api` be the default in some future release as @anthonyharrison suggests.